### ISSUE TITLE
[BUG] DMCA detection stops sync from downloading files

### ISSUE THREAD

**Issue by ahydronous** [SamSchott/maestral-dropbox#69]
**Describe the bug**
I installed Maestral, syncing went mostly fine but it threw up 2 sync errors for DMCA contested files, despite those being in private folders without any shared links or e-mails

**To Reproduce**
Put a DMCA contested file (the GBA bios would be an example) in a private folder on Dropbox via the official Dropbox app. Maestral will attempt to download it, throwing a sync error

**Expected behaviour**
Just sync the file

**System (please complete the following information):**
 - Maestral v0.4.3
 - OS: macOS Catalina

**Comment by SamSchott** [comment]
Unfortunately, this error is raised by the Dropbox API itself only passed on Maestral. I suspect that this is a limitation of the public API.

**Comment by ahydronous** [comment]
Well RIP, guess I'll just have to zip them. Thanks for the clarification!

**Comment by ardhipoetra** [comment]
I also got the same issue. Is it possible to block those files? Maestral always try it so it's become annoying for the long run

I can understand it if it's happen on the initial sync though

**Comment by SamSchott** [comment]
The easiest option would be to store all DCMA protected files in a separate folder and excluded it from syncing.

There is a currently undocumented config option "excluded_files" which can be used for this. To do so, you can manually edit the config file ('~/.config/maestral/maestral.ini' on Linux) and add the file paths relative to Dropbox root to the "excluded_files" list. For example:

```ini
[main]
path = /Users/samschott/Dropbox (Maestral)
default_dir_name = Dropbox (Maestral)
excluded_folders = []
excluded_files = ['/file.txt', '/folder/file.doc']
version = 10.0.0
```

You will need to restart Maestral for changes to the config file to take effect.

The reasons why this is currently undocumented:

1) If you remove files from the list, they will not automatically be re-synced. You will either need to manually edit the file to trigger a re-sync or rebuild Maestral's index.

2) Glob patterns such as in '.gitinore' are not supported.

I was thinking some time ago about introducing a '.mignore' file. There even is a branch for this, but due to lack of demand and a somewhat cumbersome implementation, I haven't gone forward with that.

**Comment by SamSchott** [comment]
Just to give a quick update: I have been able to reproduce the issue of maestral continuously retrying failed downloads. This has been fixed in v0.6.2:

1. Failed downloads will only be retried when maestral is restarted or syncing is paused and resumed.
2. You can now directly exclude files using the same GUI or CLI interface as for folders.

Hope this helps. Unfortunately, there is no way of actually allowing the download of DCMA files through the official Dropbox API.
